pages/common/lsof.md

```diff
@@ -32,6 +32,6 @@
 
 `lsof +D {{path/to/directory}}`
 
-- Find the process that is listening on a local TCP port:
+- Find the process that is listening on a local IPv4 TCP port, don't convert network numbers or port numbers:
 
-`lsof -iTCP:{{port}} -sTCP:LISTEN`
+`lsof -i4TCP:{{port}} -sTCP:LISTEN -n -P`
```

Comment:
Not really a fan of explicitly specifying IPv4 here. IPv4 will be going away eventually - we should be promoting the use and uptake of IPv6 instead.

Reply:
True, but my intention was to inform the users of the -4 (or -6) flag here. 
I believe the users will realize that they can change -4 to -6 for IPv6.

Reply:
True, but could we specify `-6` instead of `-4` here please?

Reply:
Done.